file-upload/file-upload-scala-sbt/fileupload-impl/src/main/scala/com/example/play/controllers/FileUploadController.scala

```diff
@@ -1,66 +0,0 @@
-package com.example.play.controllers
```

Comment:
This is the Scala version. Not a problem but probably should be deleted in #67 instead.